powershell-gallery/docs-conceptual/powershellget/supported-repositories.md

```diff
@@ -0,0 +1,283 @@
+---
+description: This article lists the repositories that have been tested with PowerShellGet v3 and how to configure them.
+ms.date: 06/28/2023
+title: Supported repository configurations
+---
+# Supported repository configurations
+
+The **Microsoft.PowerShell.PSResourceGet** module, also known as **PowerShellGet v3**, works with
+NuGet package repositories and local file stores. In general, the cmdlets should work with any NuGet
+repository that supports the NuGet protocol. However, not all NuGet repositories support all the
+features of PSResourceGet.
+
+The following repositories have been tested with PSResourceGet.
+
+## The PowerShell Gallery
+
+The PowerShell Gallery is the default repository for PSResourceGet. The Gallery is a NuGet
+repository that uses the NuGet v2 protocol.
+
+You don't need credentials to search, download, or install packages from the PowerShell Gallery. You
+must create an account and API key to publish packages to the PowerShell Gallery. For more
+information, see [Creating and publishing an item][01].
+
+The PowerShell Gallery supports all the features of PSResourceGet.
+
+## NuGet.org
+
+NuGet.org is a public host of NuGet packages used by millions of .NET and .NET Core developers every
+day. The NuGet.org provides Gallery a NuGet repository that uses the NuGet v3 protocol.
+
+Use the following command to register NuGet.org as a PSResource repository:
+
+```powershell
+$params = @{
+    Name = 'NuGetGallery'
+    Uri  = 'https://api.nuget.org/v3/index.json'
+}
+Register-PSResourceRepository @params
+```
+
+You don't need credentials to search, download, or install packages from NuGet.org. You must create
+an account and API key to publish packages to NuGet.org. For more information about getting a NuGet
+API key, see [Quickstart: Create and publish a NuGet package][05].
+
+### NuGet.org limitations
+
+The **Microsoft.PowerShell.PSResourceGet** module doesn't support the following scenarios for
+NuGet.org repositories:
+
+- Wildcard search by name, for example:
+  - `Find-PSResource -Name '*'`
+  - `Find-PSResource -Name 'Az*'`
+  - `Find-PSResource -Name 'Az*' -Tag compute`
+- Search by command or DSC resource name, for example:
+  - `Find-PSResource -Command commandname`
+  - `Find-PSResource -DSCResource dscresourcename`
+
+## Azure Artifacts
+
+Azure Artifacts is a feature of Azure DevOps. Azure Artifacts enables developers to share their code
+efficiently and manage all their packages from one place. With Azure Artifacts, developers can
```

Comment:
Does it allow developers to manage their packages _from_ one place or _in_ one place? Both?

Reply:
stet